plugins/vocabbuilder.koplugin/main.lua

```diff
@@ -1443,12 +1445,24 @@ local VocabBuilder = WidgetContainer:extend{
 
 function VocabBuilder:init()
     self.ui.menu:registerToMainMenu(self)
+    self:onDispatcherRegisterActions()
 end
 
 function VocabBuilder:addToMainMenu(menu_items)
     menu_items.vocabbuilder = {
         text = _("Vocabulary builder"),
         callback = function()
+            self:onShowVocabBuilder()
+        end
+    }
+end
+
+function VocabBuilder:prepareWidgetToShow()
+    if self.builder_widget then
+        self.builder_widget:reloadItems()
+    else
+        -- We initiate the widget with proper
+        -- callback definition for reload_item
             local reload_items = function(builder_widget)
                 builder_widget.reload_time = os.time()
```

Comment:
Thank you for trying to make the diff smaller :)
I had looked at some previous commit and the indentation shifts was awful for the diff).
But this indentation hiccup is noticable :)
I prefer this trick though. May be make it leaner, just double indenting the function content ?
```lua
    else
        -- We initiate the widget with proper
        -- callback definition for reload_item
        local reload_items = function(builder_widget)
                builder_widget.reload_time = os.time()
```

Reply:
Done.